test/e2e/framework/util.go

```diff
@@ -764,28 +764,28 @@ func waitForServiceAccountInNamespace(c clientset.Interface, ns, serviceAccountN
 }
 
 func WaitForPodCondition(c clientset.Interface, ns, podName, desc string, timeout time.Duration, condition podCondition) error {
-	Logf("Waiting up to %[1]v for pod %[2]s status to be %[3]s", timeout, podName, desc)
+	Logf("Waiting up to %v for pod %q in namespace %q to be %q", timeout, podName, ns, desc)
```

Comment:
When the loop ends for the pod-delete calls, Status.Reason is typically "" but sometimes I see other reasons. The `desc` value is passed in by the caller and meant to be descriptive of the condition AFAICT. I'm creating a new pr (will update with #) to deal with better detection of pod deletion and in this pr the reason parm will not be checked if it is "".